packages/itwin/geo-tools/src/UiProvider.tsx

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import type {
+  CommonToolbarItem, UiItemsProvider,
+} from "@itwin/appui-abstract";
+import {
+  StageUsage, ToolbarOrientation, ToolbarUsage,
+} from "@itwin/appui-abstract";
+import { ToolbarHelper } from "@itwin/appui-react";
+import { GeoToolsItemDef } from "./GeoToolsItemDef";
+
+export class GeoToolsUiItemsProvider implements UiItemsProvider {
```

Comment:
Shouldnt we call this, GeoAddressSearchItemsProvider.  I see GeoTools as a collection of tools, GeoAddressSearch being the first and only one for now....

Reply:
Yea that sounds good with me, i was just playing around with your code locally, and needed a ui provider to add it to my viewer

Reply:
will re-add in separate pr